data/hai/hai reveal 5 philosophers.txt

```diff
@@ -485,8 +485,8 @@ mission "Hai Reveal [B05-long] The Red Mystery"
 
 mission "Hai Reveal [B06-long] Tarazed Analysis"
 	landing
-	name "Seeking Aid"
-	description `Find and board a smugglers ship. Your display will identify them as the "Smuggler" government. Try Syndicate space or the Far North. Then investigate any leads and return to <planet>.`
+	name "Hunting Smugglers"
+	description `Find and board a smuggler's ship. Your display will identify them as the "Smuggler" government, but they will still look like Merchants. Try Syndicate space or the Far North. Then investigate any leads and return to <planet>.`
```

Comment:
>The fact that the behaviour hasn't always been that the Navy and co. don't attack these smuggler ships 

The behaviour, during the development phase, where some things may be considered temporary, and where we're not even concerned about preserving save files through changes... that behaviour?
I'm dismissing this argument out of hand.

>This concept that NPCs think a ship has a different government to what we are shown is entirely new to the player at this point.

This could potentially be a good point, except in FW they explicitly call you out to "repaint" your ship to display correct colours, so I don't think it's actually accurate.

Reply:
Gonna be real, dog. The "development phase" was during the PR. We're closing in on a release with this content. Now is not the time to blow off feedback because "Well, it's still in development and _may_ be temporary." Now you might be saying something different, but that just doesn't come off well to me.